lib/lhm/connection.rb

```diff
@@ -18,11 +22,6 @@ def initialize(connection:, default_log_prefix: nil, options: {})
       super(connection)
     end
 
-    def ar_connection
-      # Get object from the simple delegator
```

Comment:
I guess this comment is deleted permanently?

Reply:
Will add it back